pkg/volume/volume_info_v1.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type VolumeInfosV1 struct {
```

Comment:
In this way, we will have to use VolumeInfosV1 and VolumeInfoV1 everywhere in the code where volume info is required, and then once we want to change the version, all the code need to be updated.
Probably, we can remove these two structures and add the version field into `VolumeInfos` structure.

Reply:
The Golang JSON unmarshal can work even if any fields are not present in the structure, but if we want to work this way, we don't need the version of this structure. 
The version is introduced to work with the potential non-backward compatible changes.
By the suggested way, to me, we already lose the ability to support non-backward compatible change.

Reply:
Yes, with Golang's JSON capability, somehow the version field is not a must-have. I am fine with the either way --- having the version field or not having it. From me, having the version field helps a little bit on the data verification:

- With the version field, we can check whether a field in `VolumeInfos` is a required one, if it is required, we can fail the process.
- For example, if a field xxx is empty, if the volume info is from an old backup, we can let it go; while if it is from a new backup, we should not continue.

Reply:
If we only have a version variable, without other information, I don't think the code knows how to check which fields are mandatory. 
If it's possible some versions must have some mandatory fields, then we need to have structures to let the code know that.